apps/experiments/export.py

```diff
@@ -10,9 +10,28 @@ def _parse_tags(tags: list[Tag]) -> str:
     return ", ".join([t.name for t in tags])
 
 
+def _parse_comments(user_comments: list[UserComment]) -> str:
+    """Parses `user_comments` into a single string that looks like this:
+    <username_1>: "user 1's comment" | <username_2>: "user 2's comment" | <username_1>: "user 1's comment"
+    """
+    comment_str = ""
+    for idx, comment in enumerate(user_comments):
+        if idx > 0:
+            comment_str = f'{comment_str} | <{comment.user.username}>: "{comment.comment}"'
+        else:
+            comment_str = f'<{comment.user.username}>: "{comment.comment}"'
+    return comment_str
```

Comment:
What about this:
```python
" | ".join([str(comment) for comment in user_comments])
```

Where `comment.__str__` does the single comment formatting: `'<{self.user.username}>: "{self.comment}"'`

Reply:
I like this, yeah